modules/nodes-nodes-audit-log-basic-viewing.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-audit-log.adoc
+
+[id="nodes-nodes-audit-log-basic-viewing-{context}"]
+= Viewing the API Audit Log
+
+You can view the basic audit log.
+
+.Procedure
+
+To view the basic audit log:
+
+. View the {product-title} API server logs
+
+.. If necessary, get the name of the log you want to view:
++
+----
+$ oc --insecure-skip-tls-verify adm node-logs --role=master --path=openshift-apiserver/
+
+ip-10-0-140-97.ec2.internal audit-2019-04-09T00-12-19.834.log
+ip-10-0-140-97.ec2.internal audit-2019-04-09T11-13-00.469.log
+ip-10-0-140-97.ec2.internal audit.log
+ip-10-0-153-35.ec2.internal audit-2019-04-09T00-11-49.835.log
+ip-10-0-153-35.ec2.internal audit-2019-04-09T11-08-30.469.log
+ip-10-0-153-35.ec2.internal audit.log
+ip-10-0-170-165.ec2.internal audit-2019-04-09T00-13-00.128.log
+ip-10-0-170-165.ec2.internal audit-2019-04-09T11-10-04.082.log
+ip-10-0-170-165.ec2.internal audit.log
+----
+
+.. View the {product-title} API server log for a specific master node and timestamp or view all the logs for that master:
++
+----
+$ oc adm node-logs <log-name> --path=openshift-apiserver/audit.log
+----
++
+For example:
++
+----
+$ oc adm node-logs ip-10-0-140-97.ec2.internal audit-2019-04-08T13-09-01.227.log --path=openshift-apiserver/audit.log
```

Comment:
Should be `oc adm node-logs ip-10-0-140-97.ec2.internal audit-2019-04-08T13-09-01.227.log --path=openshift-apiserver/audit-2019-04-08T13-09-01.227.log`